crates/rattler_solve/Cargo.toml

```diff
@@ -38,7 +38,8 @@ url = { workspace = true }
 
 [features]
 default = ["resolvo"]
-libsolv_c = ["rattler_libsolv_c", "libc"]
+libsolv_c = ["dep:rattler_libsolv_c", "dep:libc"]
+resolvo_diagnostics = ["resolvo/diagnostics"]
```

Comment:
Enables `resolvo/diagnostics` when enabled, what happens if `resolvo` is disabled though?

Reply:
Changed it!